wonnx/src/gpu.rs

```diff
@@ -551,6 +589,15 @@ impl GpuStep {
                             bytemuck::cast_slice(&resize(int_input.to_vec())),
                         );
                     }
+                    InputTensor::I64(int_input) => {
+                        log::warn!("reading int64 input as int32 (int64 is not supported for calculation but can be used as input as long as values fit in int32)");
+                        let int32_input = int_input.iter().map(|x| *x as i32).collect();
```

Comment:
Similar as above

Reply:
Is this unfaillable? (Genuine question)

Reply:
> Is this unfaillable? (Genuine question)

It will wrap without warning:

````rust
let x: i64 = i64::MAX - 3;
let xx = x as i32;
println!("x={} xx={}", x, xx);
````

This will print `x=9223372036854775804 xx=-4`

Reply:
@haixuanTao you're absolutely right! I changed this to use `i32::from_i64` just like in the other places (I forgot this one for some reason).